javascript/src/main/components/Tutor/TutorTable.js

```diff
@@ -2,16 +2,16 @@ import React from "react";
 import BootstrapTable from "react-bootstrap-table-next";
 import { Button } from "react-bootstrap";
 import { useHistory } from "react-router-dom";
-import useSWR from "swr";
-import { fetchWithToken } from "main/utils/fetch";
-import { useAuth0 } from "@auth0/auth0-react";
+// import useSWR from "swr";
+// import { fetchWithToken } from "main/utils/fetch";
+// import { useAuth0 } from "@auth0/auth0-react";
 
-import { buildDeleteTutor } from "main/services/Tutor/TutorService";
+// import { buildDeleteTutor } from "main/services/Tutor/TutorService";
 
 export default ({ tutors, instructorTutors, admin, deleteTutor }) => {
   const history = useHistory();
-  const { getAccessTokenSilently: getToken } = useAuth0();
-  const { data: roleInfo } = useSWR(["/api/myRole", getToken], fetchWithToken);
+  // const { getAccessTokenSilently: getToken } = useAuth0();
+  // const { data: roleInfo } = useSWR(["/api/myRole", getToken], fetchWithToken);
```

Comment:
Should we just remove these entirely instead of commenting them out?

Reply:
not sure, i removed in most cases but the crud stuff seems really important for tutor table so just left it at comments, but if you think should be removed I can just take it out

Reply:
Ok gonna just take out since taking others out but need to keep this in mind if students are gonna work on fixing the really broken crud stuff on cs-las